app/assets/javascripts/discourse/templates/modal/keyboard-shortcuts-help.hbs

```diff
@@ -55,6 +55,23 @@
         <li>{{html-safe shortcuts.actions.quote_post}}</li>
       </ul>
     </section>
+    {{#if showBookmarkShortcuts}}
+    <section>
```

Comment:
might be worth to have a css class here, otherwise targeting anything here will be complicated